src/rapidfuzz/process_cpp_impl.pyi

```diff
@@ -101,29 +101,35 @@ UINT16: int
 UINT32: int
 UINT64: int
 
-def cdist(
-    queries: Iterable[_S1],
-    choices: Iterable[_S2],
-    *,
-    scorer: Callable[..., _ResultType] = ratio,
-    processor: Callable[..., _StringType] | None = None,
-    score_cutoff: _ResultType | None = None,
-    score_hint: _ResultType | None = None,
-    score_multiplier: _ResultType | None = None,
-    dtype: int | None = None,
-    workers: int = 1,
-    scorer_kwargs: dict[str, Any] | None = None,
-) -> Any: ...
-def cpdist(
-    queries: Iterable[_S1],
-    choices: Iterable[_S2],
-    *,
-    scorer: Callable[..., _ResultType] = ratio,
-    processor: Callable[..., _StringType] | None = None,
-    score_cutoff: _ResultType | None = None,
-    score_hint: _ResultType | None = None,
-    score_multiplier: _ResultType | None = None,
-    dtype: int | None = None,
-    workers: int = 1,
-    scorer_kwargs: dict[str, Any] | None = None,
-) -> Any: ...
+try:
+    import numpy as np
+
+    def cdist(
+        queries: Iterable[_StringType1],
+        choices: Iterable[_StringType2],
+        *,
+        scorer: Callable[..., _ResultType] = ratio,
+        processor: Callable[..., _StringType] | None = None,
+        score_cutoff: float | None = None,
+        score_hint: float | None = None,
+        score_multiplier: float | None = None,
+        dtype: int | None = None,
+        workers: int = 1,
+        scorer_kwargs: dict[str, Any] | None = None,
+    ) -> np.ndarray: ...
+    def cpdist(
+        queries: Iterable[_StringType1],
+        choices: Iterable[_StringType2],
+        *,
+        scorer: Callable[..., _ResultType] = ratio,
+        processor: Callable[..., _StringType] | None = None,
+        score_cutoff: float | None = None,
+        score_hint: float | None = None,
+        score_multiplier: float | None = None,
+        dtype: int | None = None,
+        workers: int = 1,
+        scorer_kwargs: dict[str, Any] | None = None,
+    ) -> np.ndarray: ...
+
+except ImportError:
+    pass
```

Comment:
This is actually wrong. The internal implementations return a custom matrix that can be converted to a numpy matrix. Not that this matters much, since the file will disappear anyways.